runtime/src/main/java/org/corfudb/runtime/clients/ClientResponseHandler.java

```diff
@@ -84,11 +85,18 @@ public boolean handle(ResponseMsg response, ChannelHandlerContext ctx) {
                     // For errors, we always want to completeExceptionally
                     errorHandlerMap.get(errorCase).handle(response, ctx, router);
                 } catch (Throwable e) {
+                    log.warn("Server threw exception for {} with request_id: {}",
+                            response.getPayload().getPayloadCase(),
+                            response.getHeader().getRequestId(),
+                            e);
```

Comment:
I'm wondering if it might be worth logging the message a bit differently. Right now, if a message type is not contained in either handlerMap, then no message is logged whatsoever. Maybe instead we can just have one guarded logging statement outside of the if?

Reply:
We had an offline discussion and decide to keep the warning log here and move the guarded log out of the if clause.